updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,166 @@
+name: Open a PR that prepares the policy for release
+pipelineid: "{{ .pipelineid }}"
+version: v0.97.0
+
+sources:
+  newestGitHubDraftReleaseVersion:
+    name: Extract version from newest GitHub draft release from release drafter
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      typefilter:
+        # We specifically only want the newest draft release, created with
+        # release-drafter, as its title has the prospective semver that we want
+        # to tag
+        draft: true
+      versionfilter:
+        kind: latest
+    transformers:
+      - trimprefix: "v"
+
+  newestReleaseTagVersion:
+    name: Extract newest tag from newest GitHub release
+    # used for displaying a gh compare URL
+    # disablesourceinput: true # we continue on fail (we may not have a tag)
```

Comment:
The `newestReleaseTagVersion` source will fail if the repo doesn't have a tag. I found no way to continue on fail (disablesourceinput would be for targets). Later on, we consume a default for the `newestReleaseTagVersion` just in case already.

Reply:
Do you have an example of such repository?